specification/figures/xss-example-human-text.md

```diff
@@ -1,7 +1,7 @@
 # Human readable text based on XSS-Example.pdf
 
 This displays how human readable descriptions can be derived from a vulntological representation of a vulnerability. <br />
-This is a representation of the XSS-Example provided at [XSS-Example.pdf](xss-example.pdf)<br />
+This is a representation of the XSS-Example provided at [XSS-Example](xss-example.png) and embedded below.<br />
```

Comment:
If the image is embedded, why not start with the image, and then follow with the explanation. This would eliminate the need for the extra link to the image.

Reply:
It makes more sense for the image to be provided second, however, there is no need to mention the image since it has been embedded further down. Removing text completely